README.md

```diff
@@ -106,7 +106,7 @@ Fetch multiple records from the datastore. Returns `[]` if the records do not ex
 * `config.paginate` - An `object`. Each of its properties further defines the characteristics for pagination
 * `config.paginate.count` - An `integer`. This is the number of items per page
 * `config.paginate.page` - An `integer`. This is the page number of the set you wish for the datastore to return
-* `callback(err, result)`  - A callback which is called when the task has succeeded. It receives the `err` and `result`. The result is always returned, with an empty array `[]` value designating that there is no item to be found.
+* `callback(err, result)`  - A callback which is called when the task has succeeded. It receives the `err` and `result`. An error is returned if the table does not exist. An empty array `[]` is returned when the table is empty.
```

Comment:
I would change this to elaborate more on the expected behavior.

Say something like:
"callback(err, result) - A callback which is called when the task has completed. It receives the 'err' and 'result'. The expected behavior is: 
1. err if the table does not exist
2. err if the call fails for any reason
3. An empty array if the table is empty
4. An array of entries equal to count

Or something like that